src/api/resolvers/qrCodeResolvers.js

```diff
@@ -0,0 +1,35 @@
+import getWeb3 from '../web3'
+export const defaults = {}
+
+const resolvers = {
+  Query: {
+    async scanQRCodeSupported() {
+      let supported = false;
+      const web3 = await getWeb3()
+      if(web3.currentProvider.scanQRCode){
+        supported = true
+      }
+      return {
+        supported:supported,
+        __typename: 'QRCodeSupported'
+      }
+    }
+  },
+  
+  Mutation: {
+    async scanQRCode() {
```

Comment:
This should not be a mutation since it doesn't change any data.

Reply:
Moved to query